classes/fields/time.php

```diff
@@ -169,6 +169,19 @@ public function is_empty( $value = null ) {
 		return empty( $value );
 	}
 
+	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$options" missing
* :no_entry_sign: Doc comment for parameter "$js" missing

<sub>hash: 7a0c357f7146e837650dee4805f8565d</sub>